pkgs/by-name/mq/mqttx-cli/package.nix

```
@@ -0,0 +1,74 @@
+{
+  fetchFromGitHub,
+  fetchYarnDeps,
+  fetchpatch,
+  lib,
+  mqttx-cli,
+  nodejs,
+  stdenv,
+  testers,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mqttx-cli";
+  version = "1.11.0";
+
+  src = fetchFromGitHub {
+    owner = "emqx";
+    repo = "MQTTX";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-IPIiSav6MPJmzetBgVw9fLGPjJ+JKS3oWMEfCJmEY84=";
+  };
+
+  patches = [
+    # moves @faker-js/faker from devDependencies to dependencies
+    # because the final package depends on it
+    # https://github.com/emqx/MQTTX/pull/1801
+    (fetchpatch {
+      url = "https://github.com/emqx/MQTTX/commit/4df2902e6f2a05f0b54dbc710128c262cb8c3201.patch";
```

```suggestion
      url = "https://github.com/emqx/MQTTX/commit/3d89c3a08477e9e2b5d83f2a222ceaa8c08e50ce.patch";
```
The commit was merged by rebase, so the id changed. You may also need to update the `hash` below

---

Thanks!
Updated the commit.